packages/block-library/src/page-list/edit.js

```diff
@@ -406,3 +407,57 @@ export default function PageListEdit( {
 		</>
 	);
 }
+export const transforms = {
+	to: [
+		{
+			type: 'block',
+			blocks: [ 'core/list' ],
+			transform: () => {
+				const pages = select( coreStore ).getEntityRecords(
+					'postType',
+					'page',
+					{
+						per_page: -1,
+						status: 'publish',
+					}
+				);
+
+				if ( ! pages || pages.length === 0 ) {
+					return createBlock( 'core/list', {
+						ordered: false,
+					} );
+				}
+
+				const createListItems = ( parentId = 0 ) => {
+					return pages
+						.filter( ( page ) => page.parent === parentId )
+						.map( ( page ) => {
+							const childItems = createListItems( page.id );
+							const listItem = createBlock( 'core/list-item', {
+								content: `<a href="${ page.link }">${ page.title.rendered }</a>`,
+							} );
+							if ( childItems.length > 0 ) {
+								const subList = createBlock(
+									'core/list',
+									{},
+									childItems
+								);
+								listItem.innerBlocks = [ subList ];
+							}
+							return listItem;
+						} );
+				};
+
+				const innerBlocks = createListItems();
+
+				return createBlock(
+					'core/list',
+					{
+						ordered: false,
+					},
```

Comment:
```suggestion
					{
						style,
						fontFamily,
					},
```

The default value of the `ordered` attribute is `false`, so it's not necessary. Instead, we need to inherit all the block support styles.

Don't forget to add the following code too:

```javascript
transform: ( attributes ) => {
	const { style, fontFamily } = attributes;
	// ...
},
```